AUTHORS

```diff
@@ -178,10 +178,10 @@ thetazero <lesha.seletskiy@gmail.com>
 Alfonso Valenciana <valencianaad@hotmail.com>
 David <dvd.cnt@gmail.com>
 Glen Whitney <glen@studioinfinity.org>
+yifanwww <yifanw1101@gmail.com>
 Divya Yeruva <divyareddy.y019@gmail.com>
 Yeruva <Divya.Yeruva@pepsico.com>
 Eternal-Rise <my_dawn@ukr.net>
-yifanwww <yifanw1101@gmail.com>
 NattapongSiri <38780353+NattapongSiri@users.noreply.github.com>
```

Comment:
Please remove a change to this file just to re-order entries, thanks.

Reply:
`npm run update-authors` re-generates the same order.

Reply:
Ok, @josdejong, does that mean we go ahead and merge this order change?

Reply:
I think there is no need to worry about the order of the names, as long as they are there. It should be ordered by first commit. Maybe the PR overrides the changes of the first commit a couple of users or so? Let's just keep the order that the script outputs and not change it by hand.